resource/keywords_bdd.robot

```diff
@@ -14,132 +14,134 @@ ${BROWSER}    firefox
-    selenium.Click Button                                                            name:submit_search
+    selenium.Input Text                                                              id=search_query_top                                                     ${product}
+    selenium.Click Button                                                            name=submit_search
 
-I should see the searched product "${product}"
-    selenium.Wait Until Page Contains Element                                        css:#center_column > h1                                           timeout=8s
-    selenium.Title Should Be                                                         Search - My Store
-    #checar se o texto "SEARCH - produto" da página de resultado confere com o produto pesquisado.
-    ${texto}                                                                         selenium.Get Text                                                 xpath://*[@id="center_column"]/h1/span[1]
-    ${textoSemEspaco}                                                                string.Strip String                                               ${texto}                                                       mode=both           characters="
-    Should Be Equal                                                                  ${textoSemEspaco}                                                 ${product}                                                     ignore_case=True
-
-    #garantir que pelo menos 1 item com a palavra procurada foi exibido.
-
-###Scenario 02: Search for non existing products
-I should see the message "No results were found for your search "${no_product}""
-    selenium.Wait Until Page Contains Element                                        css:#center_column > h1
-    #verifica se a mensagem de "no results were found" é exibido na página.
-    selenium.Element Should Contain                                                  xpath://*[@id="center_column"]/p[@class="alert alert-warning"]    ${no_product}
-
-
-###Scenario 03: List products
 I select the "WOMEN" category in the main menu categories
-    selenium.Mouse Over                                                              css:#block_top_menu > ul > li:nth-child(1) > a
+    selenium.Mouse Over                                                              css=#block_top_menu > ul > li:nth-child(1) > a
 
 I select the sub-category "${product}"
-    selenium.Wait Until Element is Visible                                           xpath://*[@id="block_top_menu"]/ul/li[1]/ul                       timeout=8s
-    selenium.Click Element                                                           link:${product}
+    selenium.Wait Until Element is Visible                                           xpath=//*[@id="block_top_menu"]/ul/li[1]/ul                             timeout=12s
```

Comment:
What do think about configure a new default timeout to entire test case?

Reply:
Done.